chain/actors/manifest.go

```diff
@@ -113,6 +113,22 @@ func ReadManifest(ctx context.Context, store cbor.IpldStore, mfCid cid.Cid) (map
 	return metadata, nil
 }
 
+// Given a Manifest CID (NOT the ManifestData CID), load the underlying ManifestData
+func LoadManifestData(ctx context.Context, mfCid cid.Cid, adtStore adt.Store) (*manifest.ManifestData, error) {
+	var mf manifest.Manifest
+	var mfData manifest.ManifestData
+
+	if err := adtStore.Get(ctx, mfCid, &mf); err != nil {
+		return nil, xerrors.Errorf("error reading manifest: %w", err)
+	}
+
+	if err := adtStore.Get(ctx, mf.Data, &mfData); err != nil {
+		return nil, xerrors.Errorf("error fetching data: %w", err)
+	}
+
+	return &mfData, nil
+}
```

Comment:
This assumes a single manifest version. Maybe that's fine?

The correct way to do this is:

1. Load the `manifest.Manifest`.
2. Call `mf.Load(ctx, adtStore)`.
3. Return the `manifest.Manifest`.

Reply:
(and probably add a method to `manifest.Manifest` for enumerating the actors)

(and yeah, I'm not a fan of this API)

Reply:
I agree -- I'm gonna drop this method for now, and use the builtin actors keys to ennumerate (and create an issue for the flow you're describing, because I agree it's needed).